cli/src/commands.rs

```diff
@@ -54,6 +54,13 @@ pub(crate) enum Command {
         /// Amount the payer will send, in satoshi
         payer_amount_sat: u64,
     },
+    /// Generates an URL to buy bitcoin from a 3rd party provider
+    BuyBitcoin {
+        provider: BuyBitcoinProvider,
```

Comment:
Does the provider have to be specified? (i.e. can't we default to Moonpay for now?)

And if so, what's the correct way to specify it in the CLI?

Reply:
I wanted to keep the same approach as the Greenlight SDK, with the addition of providing the amount to buy. In the CLI we first call `prepare_buy_bitcoin()` to validate the amount limits and the amount is locked in the request to Moonpay.
```
breez-liquid-cli> buy-bitcoin moonpay 100000
```